doc/config_manager.rst

```diff
@@ -26,15 +26,15 @@ repositories are enabled or disabled, and add new repositories.
 Synopsis
 --------
 
-``dnf config-manager [options] <repo>...``
+``dnf config-manager [options] <repoid>...``
 
 ---------
 Arguments
 ---------
 
-``<repo>``
-    Display / modify specified repository. If not specified display / modify main DNF configuration.
-    Repositories can be specified using globs.
+``<repoid>``
+    Display / modify specified repository identified by <repoid>. If not specified, display / modify
+    main DNF     configuration. Repositories can be specified using globs.
```

Comment:
Some excessive spaces creeped in...

Also, _... specified repository identified by <repoid> ..._ doesn't really make sense, sorry I missed that in my previous suggestion. I'd go with _... a repository identified by <repoid> ..._, see also the next comment...